force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -320,15 +414,21 @@ public with sharing class ClassesObjectsMethods {
     public static List<String> personDetails() {
         List<String> results = new List<String>();
         // Create a new instance of Person class
+        Person person1 = new Person();
         // Name the person 'John Doe' and set the age to 28
-
+        person1.setName('John Doe');
+        person1.setAge(28);
         // Add the details of the person using the introduceYourself method
 
+        results.add(person1.introduceYourself());
+
         // Get the name of the person using the getter method and add it to the list
+        results.add(person1.getName());
 
         // Get the age of the person using the getter method and add it to the list
-        
-        return null; // Replace null with the variable you used to store the result
+        results.add(person1.getAge().toString());
+
+        return results; // Replace null with the variable you used to store the result
```

Comment:
easier if you had a constructor to use 

 Person person = new Person('John Doe', 28);

Reply:
Agreed.   Having an overloaded constructor in Person class would avoid some lines in this method.